core/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
 
   implementation 'org.apache.tuweni:tuweni-bytes'
   implementation 'org.apache.tuweni:tuweni-net'
+  implementation 'org.web3j:core'
```

Comment:
Do we need to use web3j in core eth2signer code now?

Reply:
We do still need Web3j as we use the "Credential Signer" still